### PR TITLE
installing in a disabled state works for windows hosts

### PR DESCRIPTION
**Pull request by blofeldthefish**
The win_service module does not have an "enabled" option, it's equivalent is called "start_mode".

To identify relevant steps in playbook, "(windows)" has been appended to task names which were the same between linux/windows to make them unique.

Added a condition so that the correct "start_mode" is configured for Windows, using same role config settings.

### CODE REVIEW

File: tasks/install_dist_pkgs.yml [hunk 1]
```diff
@@ -133,11 +133,11 @@
     - nrinfragent_state != "absent"
     - nrinfragent_os_name|lower != 'windows'
 
-- name: setup agent service
+- name: setup agent service (windows)
   win_service:
     name: newrelic-infra
     state: "{{ nrinfragent_service_state }}"
-    enabled: "{{ nrinfragent_service_enabled }}"
+    start_mode: "{{ ( when nrinfragent_service_enabled ) | ternary('auto','manual') }}"
```
Comment (csmartins) [inline tasks/install_dist_pkgs.yml:140]: Experienced that same problem trying to install on some gcp compute windows instances and fixed with the code below.

```suggestion
    start_mode: "{{ nrinfragent_service_enabled | ternary('auto','manual') }}"
```

**Comment by csmartins** [comment]
@josemore do you hold ownership for this repository? If you do, can you review this, please?

**Comment by jharris-** [comment]
Any update one this PR?
Looks like the travis builders are failing due to not having docker available...
FWIW, this change is also for windows only and all of the travis test builders are linux so the changes will not be tested anyway

**Comment by jjensen-incomm** [comment]
Just adding a comment to see if this can get any attention. We are essentially unable to make use of any Ansible automation to deploy our NewRelic infrastructure agent against our Windows hosts due to this issue.

**Comment by davetapley** [comment]
@jjensen-incomm FYI I have a fix [here](https://github.com/JEFuller/infrastructure-agent-ansible/commit/d383bd518f070a51526f82eddb5b7c14a1137a0f) which you're welcome to use. 

Just put this ⬇️ in a `requirements.yml` and run `ansible-galaxy install -r requirements.yml`.

```yml
---
roles:
  - src: https://github.com/JEFuller/infrastructure-agent-ansible
    version: main
    name: newrelic.newrelic-infra
```

**Comment by davetapley** [comment]
@csmartins @jjensen-incomm I opened this to see if we can get some 👀 from New Relic on here:
https://discuss.newrelic.com/t/infrastructure-agent-ansible-is-unmaintained-needs-attention/147403

**Comment by jjensen-incomm** [comment]
Thanks for your efforts @davetapley , I appreciate it. I reached out to our NewRelic reps for our business as well to see if they can poke anyone on their side to get this moving again.

**Comment by josemore** [comment]
Thanks for the PR, we are planning a new strategy that help us support and maintain the ansible/puppet/chef modules and expand them to all New Relic capabilities and other configuration management tools. 
The CI pipelines need a considerable maintenance effort and the team is focused on other initiatives.

**Review by varas — APPROVED** [review]

**Pull request merged**